openmetadata-service/src/main/java/org/openmetadata/service/jdbi3/APIServiceRepository.java

```diff
@@ -17,4 +18,17 @@ public APIServiceRepository() {
         ServiceType.API);
     supportsSearch = true;
   }
+
+  @Override
+  public void storeRelationships(ApiService service) {
```

Comment:
not sure eh? but could this live under `ServiceEntityRepository` instead of in all the service repos?

Reply:
oki understood, thanks